src/interactivity/base/InteractivityFactory.hpp

```diff
@@ -40,5 +69,8 @@ namespace Microsoft::Console::Interactivity
 
     private:
         void _WritePseudoWindowCallback(bool showOrHide);
+
+        HWND _pseudoConsoleWindowHwnd{ nullptr };
+        WRL::ComPtr<PseudoConsoleWindowAccessibilityProvider> _pUiaProvider{ nullptr };

```

Comment:
nit: plz rename this to be more specific to the pseudoconsole; right now `InteractivityFactory` is _pretty generic_ and `pUiaProvider` might give consumers the wrong impression